shared/services/config/nethermind-params.go

```diff
@@ -128,6 +131,17 @@ func NewNethermindConfig(cfg *RocketPoolConfig) *NethermindConfig {
 			OverwriteOnUpgrade: false,
 		},
 
+		FullPruningMaxDegreeOfParallelism: config.Parameter{
+			ID:                 "fullPruningMaxDegreeOfParallelism",
+			Name:               "Full pruning parallelism",
+			Description:        "The number of threads allocated to concurrently by Nethermind to prune data.",
```

Comment:
The description is a bit misleading as the option will determine the number of threads but is not directly that number. e.g. the 0 option doesn't mean 0 threads.